### PR TITLE
[LIB-371] Base object init

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

File: src/syncano.js [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
-const Syncano = stampit();
-
-models.Instance.please().list().then((response) => {
-    response.objects[0].delete();
-    response.objects[0].save();
-})
+const Syncano = stampit()
+  .init(({instance}) => {
+    _.forEach(models, (model, name) => {
+      instance[name] = stampit().compose(model).setBaseObject(this)();
```
Comment (dancio) [inline src/syncano.js]: u don't need those last `()`

File: src/syncano.js [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
+    });
+  })
+  .refs({
+    baseUrl: 'https://api.syncano.io',
+    accountKey: ''
+  })
+  .methods({
+    setKey(key) {
+      if(_.isEmpty(key)) throw Error('Key is required');
+      this.connection.accountKey = key;
```
Comment (dancio) [inline src/syncano.js]: u don't have `connection`

File: src/syncano.js (hunk 1 above)
Comment (jakubbilko) [inline src/syncano.js]: If you don't use the `()` then syncano.Instance will return the factory, so you'll need to use it as `syncano.Instance()` ;)

File: src/syncano.js (hunk 2 above)
Comment (jakubbilko) [inline src/syncano.js]: Ok, I thought we're dropping the `connection` object ;)

Comment (jakubbilko) [inline src/syncano.js]: 856993bfbdc15011af537bced9f6ff4e71fc657a

**Comment by dancio** [comment]
lgtm

**Pull request merged**